docs/source/specs/openapi.json

```diff
@@ -3608,6 +3620,64 @@
                     }
                 }
             },
+            "TagRate": {
+                "required": [
+                    "metric"
+                ],
+                "properties": {
+                    "tag_key": {
+                        "type": "string",
+                        "enum": ["key_one"]
```

Comment:
What does `enum` mean here?

Reply:
enum is short for "enumerate" and is a data type that consists of predefined values. Essentially it is just part of the openapi spec syntax.